libsql/src/rows.rs

```diff
@@ -74,6 +75,18 @@ impl Rows {
     pub fn column_type(&self, idx: i32) -> Result<ValueType> {
         self.inner.column_type(idx)
     }
+
+    /// Converts current [Rows] into asynchronous stream, fetching rows
+    /// one by one. This stream can be further used with [futures::StreamExt]
+    /// operators.
+    #[cfg(feature = "stream")]
+    pub fn into_stream(mut self) -> impl futures::Stream<Item = Result<Row>> + Unpin {
+        Box::pin(async_stream::try_stream! {
+            if let Some(row) = self.next().await? {
+                yield row
+            }
+        })
+    }
```

Comment:
I don't think it's necessary to `Box::pin` the stream and require `Unpin`. The caller can choose to either box it, of simply `tokio::pin!()` it to the stack if that's what they want